sig-instrumentation/annual-report-2022.md

```diff
@@ -0,0 +1,150 @@
+   If one of those ways is more full time support, what would they work on and why?
+
+   - Yes
+   -
+
+## Membership
+
+- Primary slack channel member count: 1,740
+- Primary mailing list member count: 324
+- Primary meeting attendee count (estimated, if needed): 8-12
+- Primary meeting participant count (estimated, if needed): 4-6
+- Unique reviewers for SIG-owned packages: 24 <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Unique approvers for SIG-owned packages: 22 <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+
+Include any other ways you measure group membership
+
+- We track active members in the SIG primarily based on devstats and meeting participation, and maintain an up-to-date roster of members in [kubernetes/org](https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-instrumentation/teams.yaml). These teams also serve as aliases for GitHub pings.
+
+## Subprojects
+
+<!--
+In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
+Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
+-->
+
+New in 2021:
+- [instrumentation](https://github.com/kubernetes-sigs/instrumentation)
+- [instrumentation-addons](https://github.com/kubernetes-sigs/instrumentation-addons)
+
+Retired in 2021:
+- [heapster](https://github.com/kubernetes-retired/heapster)
+- [mutating-trace-admission-controller](https://github.com/kubernetes-retired/mutating-trace-admission-controller)
+
+Continuing:
+- [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics)
+- [metrics](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/metrics)
+- [custom-metrics-apiserver](https://github.com/kubernetes-sigs/custom-metrics-apiserver)
+- [metrics-server](https://github.com/kubernetes-sigs/metrics-server)
+- [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter)
+- [klog](https://github.com/kubernetes/klog)
+- [structured-logging](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/logs)
+- [metric-stability-framework](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/component-base/metrics)
+- [instrumentation-tools](https://github.com/kubernetes-sigs/instrumentation-tools)
+
+## Working groups
+
+<!--
+In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
+Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
+-->
+
+New in 2021:
+- [WG Structured Logging](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
+
+## Operational
+
+Operational tasks in [sig-governance.md]:
+
+- [ ] [README.md] reviewed for accuracy and updated if needed
```

Comment:
I owe a small update which needs to be approved by steering.

Reply:
is this done?

Reply:
Yes: https://github.com/kubernetes/community/pull/6464